src/lib/annotations/image/ImageAnnotator.js

```diff
@@ -89,6 +89,13 @@ class ImageAnnotator extends Annotator {
      */
     createAnnotationThread(annotations, location, type) {
         let thread;
+
+        // Corrects any image annotation page number to 1 instead of -1
+        const fixedLocation = location;
```

Comment:
@JustinHoldstock this should cover anyone who created an image annotation before this commit was merged in